File: MPI/mpicheck/tests/functions/2.0/MPI_Request_get_status.f

        
        program main
        include 'mpif.h'
        
        INTEGER var_0
       LOGICAL var_1
       INTEGER var_2(10)
       INTEGER var_3
        call mpi_request_get_status(var_0, var_1, var_2, var_3)
       call pmpi_request_get_status(var_0, var_1, var_2, var_3)
        end program main
    

File: MPI/mpicheck/tests/functions/2.0/MPI_Request_get_status.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Request var_0;
    int *var_1;
    MPI_Status *var_2;
    int ret;
    /* calls */
    ret = MPI_Request_get_status(var_0, var_1, var_2);
    ret = PMPI_Request_get_status(var_0, var_1, var_2);
    return 0;
}
